emeritus-tech/landing-pages-libraries
# [CPT-432] Wharton HR template

🎟[ Jira Ticket CPT-522](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-432)

🚢 Deployment Instructions

N/A

✅ QA instructions

** Testimonial slider should work as expected using CSS only (not entirely the same as the original landing page one)

*  Run command npm start
*  Open http://localhost:8080/WHR.html
*  And verify on browser:
    *  Design should look exactly as original [landing page](https://execed-emeritus.wharton.upenn.edu/hr-management/index.php).

🖼 Previews

![screencapture-localhost-8080-WHR-html-2020-05-29-17_18_14](https://user-images.githubusercontent.com/58177278/83302096-1e7d8200-a1d1-11ea-9664-35cc65690edc.png)
